modules.json

```diff
@@ -2,60 +2,17 @@
     "name": "nf-core/isoseq",
     "homePage": "https://github.com/nf-core/isoseq",
     "repos": {
-        "nf-core/modules": {
-            "git_url": "https://github.com/nf-core/modules.git",
+        "https://github.com/nf-core/modules.git": {
```

Comment:
Think there is a problem with this file, I think you probably took it from the template directly

Reply:
I reinstalled all modules to fix the modules.json file. 
Thanks for pointing this issue.